### PR TITLE
[#168406196] upgrade code to work with python 3.6+

### PR DESCRIPTION
**Pull request by uraniumanchor**
# Contributing to the Donation Tracker

First of all, thank you for taking the time to contribute!

Please fill out the template below and check the following boxes:

- [X] I've added tests or modified existing tests for the change.
- [X] I've humanly end-to-end tested the change by running an instance of the tracker.

### Issue from Pivotal Tracker

https://www.pivotaltracker.com/story/show/168406196

### Description of the Change

Updating all the code to work with Python 3.6+.

### Possible Drawbacks

Our unit tests still don't cover everything so there's some possibility of something obscure that got missed.

### Verification Process

Lots of manual testing of things that I know aren't fully covered by unit tests.

### CODE REVIEW

File: eventutil.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import json
 import traceback
-import urllib2
+import urllib.request, urllib.error, urllib.parse
```
Comment (faultyserver) [inline eventutil.py]: `error` and `parse` look unused? Based on the diff at least

File: management/commands/cache_giantbomb_info.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import sys
 import time
 import json
-import urllib
-import urllib2
+import urllib.request, urllib.parse, urllib.error
+import urllib.request, urllib.error, urllib.parse
```
Comment (faultyserver) [inline management/commands/cache_giantbomb_info.py]: imported twice? `urllib.error` is also unused.

File: templatetags/donation_tags.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 import datetime
 import locale
-import urllib
+import urllib.request, urllib.parse, urllib.error
```
Comment (faultyserver) [inline templatetags/donation_tags.py]: `request` and `error` unused?

File: admin.py [hunk 4]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (justinkim) [inline admin.py]: whomst

File: prizeutil.py [hunk 5]
```diff
@@ -84,8 +85,8 @@ def get_past_due_prize_winners(event):
 def close_past_due_prize_winners(event, verbosity=0, dry_run=False):
     for prizewinner in get_past_due_prize_winners(event):
         if verbosity > 0:
-            print("Closing Prize Winner #{0} with {1} pending".format(
-                prizewinner.id, prizewinner.pendingcount))
+            print(("Closing Prize Winner #{0} with {1} pending".format(
```
Comment (justinkim) [inline prizeutil.py]: Are the added parens needed?

File: tests/test_eventutil.py [hunk 6]
```diff
@@ -46,6 +46,7 @@ def test_request_made(self):
         assert responses.calls[0].request.url == 'https://example.com/?comment=&amount=1.5&timereceived=2018-01-01+00%3A00%3A00&donor__visibility=FIRST&domain=PAYPAL&id=1&donor__visiblename=%28No+Name%29'
         assert responses.calls[0].response.status_code == 200
 
+    @skip("still valid in Python 3?")
```
Comment (justinkim) [inline tests/test_eventutil.py]: What failure were you getting?

File: volunteer.py [hunk 7]
```diff
@@ -97,11 +97,11 @@ def send_volunteer_mail(domain, event, volunteers, template, sender=None, token_
 
                 if verbosity > 0:
                     if created:
-                        print("Created user {0} with email {1}".format(
-                            volunteer.username, volunteer.email))
+                        print(("Created user {0} with email {1}".format(
```
Comment (justinkim) [inline volunteer.py]: Same with these parens?

File: tests/test_eventutil.py (hunk 6 above)
Comment (uraniumanchor) [reply tests/test_eventutil.py]: The error case doesn't trigger, I'm not sure it even CAN any more, so I'm not sure the test is useful.

**Pull request merged**